Content.Server/Buckle/Systems/StrapSystem.cs

```diff
@@ -19,6 +23,30 @@ public override void Initialize()
             base.Initialize();
 
             SubscribeLocalEvent<StrapComponent, GetInteractionVerbsEvent>(AddStrapVerbs);
+            SubscribeLocalEvent<StrapComponent, ContainerGettingInsertedAttemptEvent>(OnInsertAttempt);
+        }
+
+        private void OnInsertAttempt(EntityUid uid, StrapComponent component, ContainerGettingInsertedAttemptEvent args)
+        {
+            // This strap is being inserted into a container. In general this is fine, But maybe, just maybe, there is a
+            // player strapped to some object that is somehow being bugged into a backpack.
+
+            // Firstly, check if the container is a backpack/storage entity?
+            if (!HasComp<SharedStorageComponent>(args.Container.Owner))
+                return;
+
+            // Ok, so we will check if any of the entities we contain is a mob, and if it is, we cancel this attempt. In
+            // general, nesting strap-entities inside of a storage is rare. This could happen for folded up roller-beds,
+            // but those should have no entities strapped to them anyways
+
+            foreach (var ent in component.BuckledEntities)
+            {
+                if (HasComp<MobStateComponent>(ent) || HasComp<SharedBodyComponent>(ent))
+                {
+                    args.Cancel();
+                    return;
+                }
+            }
```

Comment:
IMO just block any insertions if anything is buckled at all.

Reply:
Block insertions into anything, or just storage?